terminologies/technique/spatialTransformation.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/technique/spatialTransformation",

```

Comment:
we should also provide the more specific variations. that one should be tasked to Timo